Filter: Unaccented string filter for char fields.

Rup-Narayan-Rajbanshi opened the pull request:
## Changes
- Add unaccented string filter for char fields.


## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [ ] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

Rup-Narayan-Rajbanshi committed:
chore(filter): add unaccented string filter for char fields.

Rup-Narayan-Rajbanshi committed:
chore(filter): add decorator for unaccented string filter

tnagorra commented on main/settings.py:
```diff
@@ -163,6 +163,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.gis",
+    "django.contrib.postgres",
```
Do we need to add postgres here? cc: @thenav56

tnagorra reviewed (commented):
For contribtutor user and user groups, we should filter on generated field.
For the rest, we can use the current approach.

susilnem reviewed (dismissed):
Let's update the test case for the changed fields

tnagorra committed:
refactor(filter): replace unaccented_filter decorator with simple func

- add support for alias in unaccented_filter
- use unaccented_filter for project name

tnagorra committed:
fix(filter)!: fix typo on usergroup filter
